systemtests/src/test/java/io/enmasse/systemtest/isolated/CommonTest.java

```diff
@@ -425,6 +452,258 @@ void testMessagingDuringRestartComponents() throws Exception {
         }
     }
 
+    @Test
+    @Tag(NON_PR)
+    @RepeatedTest(value = 3, name = RepeatedTest.LONG_DISPLAY_NAME)
```

Comment:
Why do we chose to repeat this test?  I notice this is not something we generally do.  What make this one special?

Reply:
I added that annotation here to make sure this test wasn't flaky, the same applies for the test "...WithRetry".
Anyway we may conclude to disable or remove the test "testFullBrokerRemainsOperableStandard" because of being a known defect so I can remove the annotations for repeating test runs at any moment

Reply:
I'm not sure it is fair to say there is a Broker defect.  Presumably, internally, the Broker is having to (asynchronously) remove messages that were on the first queue, freeing up space, and then reset internal state to reflect the fact that it is no longer full.  I don't think it would be reasonable to expect this to be synchronous with the delete of the first queue (after all the queue could be enormous).   So for me, having to wait a period before the colocated queue accepts messages seems fine.

I think we should remove testFullBrokerRemainsOperableStandard and keep only the version that retries.

Reply:
@k-wall wdyt about "testFullBrokerRemainsOperableStandardCreateSecondAddressAfter" does that test make sense? should it be removed as well?

Reply:
I think I expect scenarios such as testFullBrokerRemainsOperableStandardCreateSecondAddressAfter to be part of the Broker test suite, rather than this one.   I'd be happy to see it removed.